rmgpy/data/thermo.py

```diff
@@ -1702,6 +1702,16 @@ def get_thermo_data_from_groups(self, species):
 
         species.molecule = [species.molecule[ind] for ind in indices]
 
+        m = 0        
+        #Pushes all charged species to the back of the list, as a result the lowest enthalpy non-charged resonance form is used for GAV estimation
+        for i in range(len(species.molecule)):
+            if species.molecule[i-m].has_Charge() is True:
+                species.molecule.append(species.molecule[i-m])
+                thermo.append(thermo[i-m])
+                del species.molecule[i-m]
+                del thermo[i-m]
+                m += 1
```

Comment:
I would suggest updating the `prioritize_thermo` method instead, similar to what I did in 85235dd06d7ec7655957bd11a009bcd06f3b3412. You can add `has_charge` as another value in the sorting algorithm.